langs/queries/javascript/highlights.scm

```diff
@@ -4,7 +4,7 @@
  (.match? @constant "^[A-Z_][A-Z_\\d]*$"))
 
 ((shorthand_property_identifier) @constant
- (.match? @constant "^[A-Z_][A-Z_\\d]*$"))
```

Comment:
This rule is intended for the convention of using ALL-CAPS for constants.

It would be more consistent to highlight `shorthand_property_identifier` as a `property.definition` instead.